### PR TITLE
trivial: Don't fail to start the daemon or fwupdtool if fwupd.conf is missing

### PR DESCRIPTION
**Pull request by superm1**
I don't see a reason it needs to be compulsory to have the daemon.conf in place, fwupd and fwupdtool are both setup to adopt sane defaults without it there.

### CODE REVIEW

File: src/fu-config.c [hunk 1]
```diff
@@ -416,53 +416,57 @@ fu_config_load (FuConfig *self, GError **error)
...
-
-	/* set up a notify watch */
-	file = g_file_new_for_path (config_file);
-	monitor = g_file_monitor (file, G_FILE_MONITOR_NONE, NULL, error);
-	if (monitor == NULL)
-		return FALSE;
-	g_signal_connect (monitor, "changed",
-			  G_CALLBACK (fu_config_monitor_changed_cb), self);
-	g_ptr_array_add (self->monitors, monitor);
+	if (g_file_test (config_file, G_FILE_TEST_EXISTS)) {
```
Comment (hughsie) [inline src/fu-config.c]: This is too deeply indented, can you push the contents of this to a `fu_config_load_from_file()` please.

**Comment by hughsie** [comment]
>  if fwupd.conf is missing

you mean if `daemon.conf` is missing I think

File: src/fu-config.c (hunk 1 above)
Comment (superm1) [reply src/fu-config.c]: Fixed

**Comment by hughsie** [comment]
LGTM, thanks.

**Pull request merged**